packages/11ty/_plugins/components/index.js

```diff
@@ -11,6 +12,14 @@ const components = require('../../_includes/components')
  */
 module.exports = function(eleventyConfig, options) {
   for (const component in components) {
-    addComponentTag(eleventyConfig, components[component], `${component}`)
+    switch(component) {
+      default:
+        addComponentTag(eleventyConfig, components[component], `${component}`)
+        break;
+      case 'lightbox':
+      case 'modal':
+        addShortcode(eleventyConfig, components[component], `${component}`)
+        break;
+    }
```

Comment:
who doesn't love `switch` case fallthrough? -- this shouldn't be necessary if we can get access to `page` data in `addComponentTag` in the future, but it's here for now